# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1287426783 14400
#      Mon Oct 18 14:33:03 2010 -0400
# Node ID 8ea4cf59c41bc340a6940504b4dbda14523b4db6
# Parent  84ad75921e3549047e1166c97bac7bbbc874e162
list deprecated and removed functions in NEWS file

diff --git a/NEWS b/NEWS
--- a/NEWS
+++ b/NEWS
@@ -1,9 +1,9 @@
-Summary of important user-visible changes for version 3.3:
+Summary of important user-visible changes for version 3.4:
 ---------------------------------------------------------
 
  ** The fltk graphics backend now implements the following callback
     properties: keypressfcn, keyreleasefcn, windowbuttondownfcn,
     windowbuttonmotionfcn, windowbuttonupfcn, and currentpoint.  These
     enable keyboard and mouse interaction, an example of which can be
     seen in the __fltk_ginput__.m script.
 
@@ -312,25 +312,25 @@ Summary of important user-visible change
   ** Two new formats are available for displaying numbers:
 
        format short eng
        format long eng
 
      Both display numbers in engineering notation, i.e., mantissa +
      exponent where the exponent is a multiple of 3.
 
-  ** The following new functions were added:
+  ** The following functions are new in Octave 3.4:
 
-      accumdim     erfcx        luupdate     ppder        rsf2csf
-      bitpack      fileread     merge        ppint        sizemax
-      bitunpack    fminbnd      nfields      ppjumps      strread
-      blkmm        fskipl       nth_element  pqpnonneg    textread
-      cbrt         ifelse       onCleanup    randi        whitebg
-      chop         ishermitian  pbaspect     repelems     
-      daspect      isindex      powerset     reset
+       accumdim   erfcx        luupdate     ppder      rsf2csf
+       bitpack    fileread     merge        ppint      sizemax
+       bitunpack  fminbnd      nfields      ppjumps    strread
+       blkmm      fskipl       nth_element  pqpnonneg  textread
+       cbrt       ifelse       onCleanup    randi      whitebg
+       chop       ishermitian  pbaspect     repelems
+       daspect    isindex      powerset     reset
 
   ** Using the image function to view images with external programs such
      as display, xv, and xloadimage is no longer supported.  The
      image_viewer function has also been removed.
 
   ** The behavior of struct assignments to non-struct values has been
      changed.  Previously, it was possible to overwrite an arbitrary
      value:
@@ -353,16 +353,73 @@ Summary of important user-visible change
      values.  Example:
      
        f = @(x) sum (x, 1);
 
      When f is called, the call is forwarded to @sum with the constant 1
      appended, and the anonymous function call does not occur on the
      call stack.
 
+ ** Deprecated functions.
+
+    The following functions were deprecated in Octave 3.0 and have been
+    removed from Octave 3.4.
+                                           
+      beta_cdf         geometric_pdf        pascal_pdf      
+      beta_inv         geometric_rnd        pascal_rnd      
+      beta_pdf         hypergeometric_cdf   poisson_cdf     
+      beta_rnd         hypergeometric_inv   poisson_inv     
+      binomial_cdf     hypergeometric_pdf   poisson_pdf     
+      binomial_inv     hypergeometric_rnd   poisson_rnd     
+      binomial_pdf     intersection         polyinteg       
+      binomial_rnd     is_bool              setstr          
+      chisquare_cdf    is_complex           struct_contains 
+      chisquare_inv    is_list              struct_elements 
+      chisquare_pdf    is_matrix            t_cdf           
+      chisquare_rnd    is_scalar            t_inv           
+      clearplot        is_square            t_pdf           
+      clg              is_stream            t_rnd           
+      com2str          is_struct            uniform_cdf     
+      exponential_cdf  is_symmetric         uniform_inv     
+      exponential_inv  is_vector            uniform_pdf     
+      exponential_pdf  isstr                uniform_rnd     
+      exponential_rnd  lognormal_cdf        weibcdf         
+      f_cdf            lognormal_inv        weibinv         
+      f_inv            lognormal_pdf        weibpdf         
+      f_pdf            lognormal_rnd        weibrnd         
+      f_rnd            meshdom              weibull_cdf     
+      gamma_cdf        normal_cdf           weibull_inv     
+      gamma_inv        normal_inv           weibull_pdf     
+      gamma_pdf        normal_pdf           weibull_rnd     
+      gamma_rnd        normal_rnd           wiener_rnd      
+      geometric_cdf    pascal_cdf
+      geometric_inv    pascal_inv
+
+    The following functions were deprecated in Octave 3.2 and will
+    be removed from Octave 3.6 (or whatever version is the second major
+    release after 3.2):
+
+      create_set          spcholinv  spmax
+      dmult               spcumprod  spmin
+      iscommand           spcumsum   spprod
+      israwcommand        spdet      spqr
+      lchol               spdiag     spsum
+      loadimage           spfind     spsumsq
+      mark_as_command     spinv      str2mat
+      mark_as_rawcommand  spkron     unmark_command
+      spatan2             splchol    unmark_rawcommand
+      spchol              split
+      spchol2inv          splu
+
+    The following functions have been deprecated in Octave 3.4 and will
+    be removed from Octave 3.8 (or whatever version is the second major
+    release after 3.4):
+
+      cellidx  fstat  values
+
 Summary of important user-visible changes for version 3.2:
 ---------------------------------------------------------
 
  ** Compatibility with Matlab graphics has been improved.
 
     The hggroup object and associated listener callback functions have
     been added allowing the inclusion of group objects.  Data sources
     have been added to these group objects such that
@@ -701,51 +758,51 @@ Summary of important user-visible change
     greatly improved by caching the help text from all functions that
     are distributed with Octave.  The pkg function has been modified
     to generate cache files for external packages when they are
     installed.
 
  ** Deprecated functions.
 
     The following functions were deprecated in Octave 3.0 and will be
-    removed in Octave 3.4 (or whatever version is the second major
+    removed from Octave 3.4 (or whatever version is the second major
     release after 3.0):
                                            
-      beta_cdf         geometric_pdf        pascal_pdf      
-      beta_inv         geometric_rnd        pascal_rnd      
-      beta_pdf         hypergeometric_cdf   poisson_cdf     
-      beta_rnd         hypergeometric_inv   poisson_inv     
-      binomial_cdf     hypergeometric_pdf   poisson_pdf     
-      binomial_inv     hypergeometric_rnd   poisson_rnd     
-      binomial_pdf     intersection         polyinteg       
-      binomial_rnd     is_bool              setstr          
-      chisquare_cdf    is_complex           struct_contains 
-      chisquare_inv    is_list              struct_elements 
-      chisquare_pdf    is_matrix            t_cdf           
-      chisquare_rnd    is_scalar            t_inv           
-      clearplot        is_square            t_pdf           
-      clg              is_stream            t_rnd           
-      com2str          is_struct            uniform_cdf     
-      exponential_cdf  is_symmetric         uniform_inv     
-      exponential_inv  is_vector            uniform_pdf     
-      exponential_pdf  isstr                uniform_rnd     
-      exponential_rnd  lognormal_cdf        weibcdf         
-      f_cdf            lognormal_inv        weibinv         
-      f_inv            lognormal_pdf        weibpdf         
-      f_pdf            lognormal_rnd        weibrnd         
-      f_rnd            meshdom              weibull_cdf     
-      gamma_cdf        normal_cdf           weibull_inv     
-      gamma_inv        normal_inv           weibull_pdf     
-      gamma_pdf        normal_pdf           weibull_rnd     
-      gamma_rnd        normal_rnd           wiener_rnd      
+      beta_cdf         geometric_pdf       pascal_pdf      
+      beta_inv         geometric_rnd       pascal_rnd      
+      beta_pdf         hypergeometric_cdf  poisson_cdf     
+      beta_rnd         hypergeometric_inv  poisson_inv     
+      binomial_cdf     hypergeometric_pdf  poisson_pdf     
+      binomial_inv     hypergeometric_rnd  poisson_rnd     
+      binomial_pdf     intersection        polyinteg       
+      binomial_rnd     is_bool             setstr          
+      chisquare_cdf    is_complex          struct_contains 
+      chisquare_inv    is_list             struct_elements 
+      chisquare_pdf    is_matrix           t_cdf           
+      chisquare_rnd    is_scalar           t_inv           
+      clearplot        is_square           t_pdf           
+      clg              is_stream           t_rnd           
+      com2str          is_struct           uniform_cdf     
+      exponential_cdf  is_symmetric        uniform_inv     
+      exponential_inv  is_vector           uniform_pdf     
+      exponential_pdf  isstr               uniform_rnd     
+      exponential_rnd  lognormal_cdf       weibcdf         
+      f_cdf            lognormal_inv       weibinv         
+      f_inv            lognormal_pdf       weibpdf         
+      f_pdf            lognormal_rnd       weibrnd         
+      f_rnd            meshdom             weibull_cdf     
+      gamma_cdf        normal_cdf          weibull_inv     
+      gamma_inv        normal_inv          weibull_pdf     
+      gamma_pdf        normal_pdf          weibull_rnd     
+      gamma_rnd        normal_rnd          wiener_rnd      
       geometric_cdf    pascal_cdf
       geometric_inv    pascal_inv
 
     The following functions are now deprecated in Octave 3.2 and will
-    be removed in Octave 3.6 (or whatever version is the second major
+    be removed from Octave 3.6 (or whatever version is the second major
     release after 3.2):
 
       create_set          spcholinv  spmax
       dmult               spcumprod  spmin
       iscommand           spcumsum   spprod
       israwcommand        spdet      spqr
       lchol               spdiag     spsum
       loadimage           spfind     spsumsq
